docs/release_notes/v1.2.0.md

```diff
@@ -0,0 +1,274 @@
+- **RESOLVED** v1.2 Release [1461](https://github.com/dapr/docs/issues/1461)
+- **RESOLVED** Documentation for Preview Features [1463](https://github.com/dapr/docs/issues/1463)
+- **RESOLVED** Document how to set the default values for actors [1470](https://github.com/dapr/docs/issues/1470)
+- **RESOLVED** [CONTENT] Document pubsub without CE for 1.2 [1474](https://github.com/dapr/docs/issues/1474)
+- **RESOLVED** Create a shareable Dapr PowerPoint deck [1477](https://github.com/dapr/docs/issues/1477)
+- **RESOLVED** [CONTENT] Add CLI reference docs for `--dapr-http-max-request-size` flags for run [1481](https://github.com/dapr/docs/issues/1481)
+- **RESOLVED** Add Docs for Actor Reentrancy [1482](https://github.com/dapr/docs/issues/1482)
+### Kit
+- Moved logger package from `dapr/dapr` to `dapr/kit` [1](https://github.com/dapr/kit/pull/1)
+
+## Upgrading to Dapr 1.2.0
+
+To upgrade to this release of Dapr, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have installed. Note that this will remove the default $HOME/.dapr directory, binaries and all containers dapr_redis, dapr_placement and dapr_zipkin. Linux users need to run sudo if docker command needs sudo:
+
+```bash
+dapr uninstall --all
+```
+
+For RC releases like this, download the latest and greatest release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+Once you have installed the CLI, run:
+
+```bash
+dapr init --runtime-version=1.2.0
+```
+
+Wait for the update to finish,  ensure you are using the latest version of Dapr(1.2.0) with:
+
+```bash
+$ dapr --version
+
+CLI version: 1.2.0
+Runtime version: 1.2.0
+```
+
+### Kubernetes
+
+#### Upgrading from previous version
+
+You can perform zero-downtime upgrades using both Helm 3 and the Dapr CLI.
+
+##### Upgrade using the CLI
+
+Download the latest RC release from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+To upgrade Dapr, run:
+
+```
+dapr upgrade --runtime-version 1.2.0 -k
+```
+
+To upgrade with high availability mode:
+
+```
+dapr upgrade --runtime-version 1.2.0 --enable-ha=true -k
+```
+
+Wait until the operation is finished and check your status with `dapr status -k`.
+
+All done!
+
+*Note: Make sure your deployments are restarted to pick the latest version of the Dapr sidecar*
+
+##### Upgrade using Helm
+
+To upgrade Dapr using Helm, run:
+
+```
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo update
+
+helm upgrade dapr dapr/dapr --version 1.2.0 --namespace=dapr-system --wait
+```
+
+Wait until the operation is finished and check your status with `dapr status -k`.
+
+All done!
+
+*Note: Make sure your deployments are restarted to pick the latest version of the Dapr sidecar*
+
+#### Starting a fresh install on a cluster
```

Comment:
How about link to a setup doc rather that document here?

Reply:
This was just generated from a template, but yes that sounds like a good idea.

Let's see if we have a suitable document to link to.

Reply:
I want to update the template to make release note more clear if possible.

Reply:
How about I link to https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/

Reply:
Just added a link to this section @daixiang0. 你觉得呢？

Reply:
A question, docs docs delay release? If not, the 1.2 docs link is better.

Reply:
By the way, the template itself is here:
https://github.com/dapr/dapr/blob/master/docs/release_notes/template.md

We can update this so all future automatically generated release notes have consistent language.

Reply:
Great idea @daixiang0. When v1.2.0 releases docs.dapr.io will point to that version and v1.1.X will be added to the list of options.

The subdomain for v1.2 already exists: https://v1-2.docs.dapr.io/. This should match the v1.2 branch of the docs repo.

We are still working on some docs issues. Generally some of the docs are still being written after the release candidate is created / code is frozen for the release. Right now all docs issues marked as related to the 1.2 release are here: https://github.com/dapr/docs/milestone/4

Keep in mind that not all of those issues may be resolved for this release.

Reply:
Cool, let's do it step by step.

Reply:
@daixiang0 I updated the links to version specific docs! This way it will also be less confusing in the future when 1.3.0 releases etc
Thanks for the suggestion!